### PR TITLE
Introduce themes category and add Classic + Refacto

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Classic is now external of the core project.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no

### CODE REVIEW

File: traces [hunk 1]
```diff
@@ -75,6 +75,11 @@ const REPOSITORIES_CATEGORIES = [
         'PrestaShop/childtheme-example',
         'PrestaShop/module-companion',
     ],
+    'themes' => [
+        'PrestaShop/classic-theme',
+        'PrestaShop/theme-refacto',
+        'PrestaShop/community-theme-16',
```
Comment (NeOMakinG) [inline traces:81]: Wut? Is this repo maintained?

Idk if we want to keep the tracing on this one so I'll wait for another review!

Comment (PrestaEdit) [reply traces:81]: Is maybe not maintened, but it's inside the list of activity trace, as contributors have work on it (like PrestaShop 1.4 :D)

**Review by NeOMakinG — APPROVED** [review]
But I think you'll see some double contrib appears since the main repository will also have the commit history of the theme!

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @PrestaEdit

**Pull request merged**